R/roxygen_extension.R

```diff
@@ -37,3 +51,45 @@ format.rd_section_editDate <- function(x, ...) {
     "\\section{Last Edit Date}{\n", x$value, "\n}\n"
   )
 }
+
+#' @export
+format.rd_section_coverage <- function(x, ...) {
+  paste0(
+    "\\section{Test Coverage}{\n", x$value, "\n}\n"
+  )
+}
+
+format_coverage_text <- function(x){
+
+  ## capture val
+  x <- tag_markdown(x)
+
+  ## capture parsed text
+  text <- trimws(x$raw)
+  test_coverage <- strsplit(text, "\n")
+
+  coverage <- lapply(
+    test_coverage,
+    function(tc){
+      if(grepl("deprecate", tc, ignore.case = TRUE)){
+        data.frame(
+          test_case = "Deprecated",
+          requirements = NA
+        )
```

Comment:
Can you give me an idea of when you would depreciate a test case/requirement? My experience with validation groups makes me think they would push back on this idea.

Reply:
For sure. So we had a case where we found a function being shared between multiple packages that we wrote. So to harming, we created a utilities package that would hold common functions. We wrote requirements and test cases for functions, so we moved them to the utilities package and deprecated them since they were no longer requirements for the original package

Reply:
Makes sense. Thanks!